website/pages/docs/k8s/helm.mdx

```diff
@@ -747,7 +747,16 @@ and consider if they're appropriate for your deployment.
     to opt-in to Connect injection. If this is true, pods can use the same annotation
     to explicitly opt-out of injection.
 
+
+  - `healthChecks` ((#v-connectinject-healthChecks)) - Synchronization of Kubernetes health probes with Consul.
```

Comment:
```suggestion
  - `healthChecks` ((#v-connectinject-healthchecks)) - Synchronization of Kubernetes health probes with Consul.
```

Reply:
We typically make these descriptions (almost) the same as the comments on helm chart values. I think we should follow the same pattern here.